docs/pages/quickstart_guide.mdx

```diff
@@ -0,0 +1,78 @@
+
+# Quick Start Guide
+The Pelican platform provides a platform that makes deploying data and accessing data easy. It provides an open-source software for federating dataset repositories together and delivering objects to computing capacity. For more information on Pelican, visit the [About Pelican](./index.mdx) page.
+
+### Assumptions
+
+Before using the Pelican client to interact with objects, this guide makes several assumptions:
+
+- You are on a computer where you have access to a terminal. The Pelican client is a command line tool.
+- You've already installed the version of Pelican appropriate for your system, and Pelican is accessible via your path. To test this on Linux, you can run
+```console
+which pelican
+```
+which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](./install.mdx) to acquire a working installation.
+
+### Useful Terminology
+**Objects:**
+Pelican should be thought of as a tool that works with federated *objects* as opposed to *files*. The reason for this is that calling something a file carries with it the connotation that the file is mutable, i.e. its contents can change without requiring a new name. Objects in a Pelican federation, however, should be treated as **immutable**, especially in any case where objects are pulled through a cache (which will be the case for almost all files in the [Open Science Data Federation](https://osg-htc.org/services/osdf), or OSDF). This is because the underlying cache mechanism, powered by [XRootD](https://xrootd.slac.stanford.edu/), will deliver whatever object it already has access to; if an object name changes at the origin, the cache will remain unaware and continue to deliver the old object. In the worst case, when the cache only has a partial object, it may attempt to combine its stale version with whatever exists at the origin. Use object names wisely!
```

Comment:
We might consider updating this a bit to include the fact that *files* carry the connotation of a filesystem (which has a lot of extra stuff like owners, permissions, etc). Pelican doesn't work with any of that.

Also, the note about the object name changing at the origin isn't true (sorry, I think I wrote that initially). If the name changes at the origin, we don't have any issues because we've created a unique object. However, if the actual *contents* of the object change without updating the name, that's when we run into issues. Can you tidy up that distinction?